### DISCUSSION TITLE
i want to use automated navigation pages , im using blazor for this BUILDER , how can i implement that?

### DISCUSSION THREAD

**Discussion by Aminenafkha1** [elsa-workflows/elsa-core#4309] [Q&A]
i want to use automated navigation pages , im using blazor for this BUILDER , how can i implement that?
for exemple i have process of activities linked to pages blazor ,
expl : SignUp Process have multiple pages?

**Comment by GeorgeS2019** [comment]
[Blazor works](https://github.com/elsa-workflows/elsa-core/issues/3810#issuecomment-1650643045)
